test/rendering/__fixtures__/scenario-color-by-expression.fix.js

```diff
@@ -302,7 +302,7 @@ export default () => ({
                       qAttrExps: {
                         qValues: [
                           {
-                            qText: "ARGB(6,118,138,70)",
+                            qText: "ARGB(255,118,138,70)",
```

Comment:
The low alpha preduced a background that was almost white, but the `isDarkColor` logic does not take alpha into account so it thinks the color is dark. Which resulted in a "white" background color with a white text color.